src/Extensions/Abstractions/BasicMachineInformation.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Sockets;
+using System.Reflection;
+using System.Threading;
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	internal class BasicMachineInformation : IMachineInformation
+	{
+		public virtual string AgentName => DefaultEmptyValue;
```

Comment:
>AgentName [](start = 24, length = 9)

Should these `public` members have comments as per the coding standards?

Reply:
The class itself is internal, so they won't be visible outside of the assembly